study-builder/studies/mbc/emails/welcome_es.html

```diff
@@ -152,7 +152,7 @@
                     <td align="left" class="padding-copy" style="padding: 20px 0 0 0; font-size: 16px; line-height: 25px; font-family: Helvetica, Arial, sans-serif; color: #666666;">Estimada/o {{participant_firstName}} {{participant_lastName}},</td>
                   </tr>
                   <tr>
-                    <td align="left" class="padding-copy" style="padding: 20px 0 0 0; font-size: 16px; line-height: 25px; font-family: Helvetica, Arial, sans-serif; color: #666666;">Gracias por haber tomado sus primeros pasos para unirse a este movimiento para transformar la investigación del cáncer!</td>
+                    <td align="left" class="padding-copy" style="padding: 20px 0 0 0; font-size: 16px; line-height: 25px; font-family: Helvetica, Arial, sans-serif; color: #666666;">Gracias por haber tomado sus primeros pasos para unirse a este movimiento para transformar la investigación del cáncer</td>
```

Comment:
Why did we remove punctuation here?

Reply:
Tania (MBC spanish translations coordinator) requested to remove it.